ansible/roles/debops.tinyproxy/defaults/main.yml

```diff
@@ -0,0 +1,501 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# debops.tinyproxy default variables
+# ==================================
+
+# .. contents:: Sections
+#    :local:
+#
+# .. include:: ../../../../includes/global.rst
+
+
+# Installation, APT packages [[[
+# ------------------------------
+
+# .. envvar:: tinyproxy__base_packages [[[
+#
+# List of APT base packages which are required by the Tinyproxy application.
+tinyproxy__base_packages: [ 'tinyproxy' ]
+
+                                                                   # ]]]
+# .. envvar:: tinyproxy__packages [[[
+#
+# List of APT packages which are required by the Tinyproxy application.
+tinyproxy__packages: []
+
+                                                                   # ]]]
+# .. envvar:: tinyproxy__allow [[[
+#
+# List of IP addresses or CIDR subnets which will be allowed to connect to the
+# Tinyproxy server in :command:`ip(6)tables` and TCP wrappers. If it's empty, remote
+# connections are not allowed.
+tinyproxy__allow: []
+
```

Comment:
I'm curious about the default use case of Tinyproxy within DebOps. With the current configuration the role configures the service as "private", where you have to specifically open the firewall for other users to access the service. But HTTP services to me are "public" by default, for example `debops.nginx` configures the webserver with this in mind. HTTP servers have additional means to restrict access, be it HTTP Basic Auth, per-IP access limits, and the like, so blocking them by default on the firewall level seems a bit excessive to me.

Of course it all depends on the use case you are aiming at with this role.